light-zk.js/tests/prover.test.ts

```diff
@@ -1,67 +1,59 @@
 import { assert, expect } from "chai";
-import { Prover } from "../src/transaction/prover";
-import { Idl } from "@coral-xyz/anchor";
-
+import { it } from "mocha";
+import { Prover } from "@lightprotocol/prover.js";
+import * as anchor from "@coral-xyz/anchor";
 let circomlibjs = require("circomlibjs");
 import { Keypair as SolanaKeypair } from "@solana/web3.js";
-import { BN } from "@coral-xyz/anchor";
```

Comment:
Since Matteos cleanup we are trying to stick to BN instead of anchor.BN.
Please stick to BN :)